common/network-types/src/session/socket/mod.rs

```diff
@@ -529,8 +558,13 @@ mod tests {
 
         let (alice, bob) = setup_alice_bob::<MTU>(network_cfg, None, None);
 
-        let mut alice_socket = SessionSocket::<MTU, _>::new_stateless("alice", alice, Default::default())?;
-        let mut bob_socket = SessionSocket::<MTU, _>::new_stateless("bob", bob, Default::default())?;
+        let sock_cfg = SessionSocketConfig {
+            frame_size: FRAME_SIZE,
+            ..Default::default()
+        };
```

Comment:
This could be extracted to an inline helper, since it's repeated multiple times without change in the tests.